README.md

```diff
@@ -23,17 +23,20 @@ It is currently expected that only the QoS/SLA tool will interact with these sum
 ## Running the docker image on Centos 7 and Ubuntu 16.04
 We recommend using the docker image to run the Accounting server and REST interface. As such, having Docker and docker-compose installed is a prerequisite.
 
-See [Ubuntu 16.04 Instructions](https://docs.docker.com/engine/installation/linux/ubuntulinux/) or [Centos 7 Instructions](https://docs.docker.com/engine/installation/linux/centos/) for details of how to install Docker.
+See [Ubuntu 16.04 Instructions](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/) or [Centos 7 Instructions](https://docs.docker.com/engine/installation/linux/docker-ce/centos/) for details of how to install Docker. There is no requirement to undertake the 'Post-installation steps for Linux'.
 
 See [Install Docker Compose](https://docs.docker.com/compose/install/) for details of how to install docker-compose
 
-1. Download the source code for the version you wish to deploy, see [here](https://github.com/indigo-dc/Accounting/releases) for a list of releases and corresponding docker image tag. The source code contains schemas and yaml files needed for deploying via docker.
+1. Download the compressed source code from [here](https://github.com/indigo-dc/Accounting/releases/latest), it does not matter where the source code is downloaded to. As the source code contains schemas and yaml files needed for deploying via docker you will need to unzip the source code and then `cd` into the inflated directory.
 
-2. Register the service as a protected resource with the Indigo Identity Access Management (IAM) service. See [here](doc/admin.md#register-the-service-as-a-protected-resource-with-the-indigo-identity-access-management-iam) for instructions.
+2. Register the service as a protected resource with the Indigo Identity Access Management (IAM) service. See [Register the service as a protected resource with the Indigo Identity Access Management (IAM)](doc/admin.md#register-the-service-as-a-protected-resource-with-the-indigo-identity-access-management-iam) for instructions, before proceeding to step 3.
```

Comment:
Bit of a wordy link when you've already used that phrase in the previous sentence. Maybe just "See [this section]..."